src/lib/Tab/LocationView/UrlsTab.php

```diff
@@ -130,21 +131,26 @@ public function renderView(array $parameters): string
 
         $customUrlAddForm = $this->createCustomUrlAddForm($location);
         $customUrlRemoveForm = $this->createCustomUrlRemoveForm($location, $customUrlPagerfanta->getCurrentPageResults());
-        $parentLocation = $this->locationService->loadLocation($location->parentLocationId);
 
         $canEditCustomUrl = $this->permissionResolver->hasAccess('content', 'urltranslator');
 
         $viewParameters = [
             'form_custom_url_add' => $customUrlAddForm->createView(),
             'form_custom_url_remove' => $customUrlRemoveForm->createView(),
-            'parent_name' => $parentLocation->contentInfo->name,
             'custom_urls_pager' => $customUrlPagerfanta,
             'custom_urls_pagination_params' => $customUrlsPaginationParams,
             'system_urls_pager' => $systemUrlPagerfanta,
             'system_urls_pagination_params' => $systemUrlsPaginationParams,
             'can_edit_custom_url' => $canEditCustomUrl,
         ];
 
+        try {
+            $parentLocation = $this->locationService->loadLocation($location->parentLocationId);
+            $viewParameters['parent_name'] = $parentLocation->contentInfo->name;
+        } catch (UnauthorizedException $exception) {
+            $viewParameters['parent_name'] = 'the parent of this location';
```

Comment:
'the parent of this location' should be translatable. We can set `$viewParameters['parent_name'] = null` and in `src/bundle/Resources/views/content/tab/url/modal_add_custom_url.html.twig:31`, if parent_name is null, we can display proper message.